app/javascript/chat/chat.jsx

```diff
@@ -1233,18 +1237,10 @@ export default class Chat extends Component {
           </div>
         );
       }
-      if (state.expanded) {
+      if (state.expanded || !state.expanded) {
```

Comment:
@benhalpern @rhymes 

So.... I know that there is definitely a better way to do this 😅

But essentially, what I'm suggesting is that we always display the search filters, regardless of whether this `state` is `expanded` or not. I think we can rely more on CSS & viewport breakpoints to determine when things are exposed on not, versus relying on doing that in the JS here.

Reply:
So...could we just remove the if conditional?

Reply:
Haha yes I would love to! As a JS noobie, I wanted to share this first before I removed anything that could be potentially really important.

Reply:
+1 on removing the if altogether 😃

Reply:
Removed it.